srcpkgs/xdaliclock/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'xdaliclock'
 pkgname=xdaliclock
 version=2.43
-revision=2
+revision=3
 build_wrksrc=X11
 build_style=gnu-configure
 makedepends="libXext-devel libXt-devel"
 short_desc="Melting digital clock"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="MIT/X11"
```

Comment:
X11

Reply:
It is not X11 too, but it is installed anyway.